<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="1000" />
  <arg name="chain_start" default="link1" />
  <arg name="chain_end" default="end" />
  <arg name="timeout" default="0.005" />

  <param name="robot_description"
    command="$(find xacro)/xacro.py '$(find hal_description)/urdf/rover_arm.xacro'" />

  <node name="ik_tests" pkg="arm_kinematics" type="ik_test" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
